index.html

```diff
@@ -0,0 +1,280 @@
+        <div class="slide-img" id="one">
+            <img src="https://images.nintendolife.com/1a6abafcbb637/botw.original.jpg" alt="img1">
+            <span>This Zelda: Breath Of The Wild Exploit Allows You To Duplicate Them <a
+                    href="https://www.nintendolife.com/news/2021/01/spirit_orb_hunters_rejoice_this_zelda_breath_of_the_wild_exploit_allows_you_to_duplicate_them">Read
+                    Full Story >>
+                </a>
+            </span>
+            <label for="i5" class="prev">◁</label>
+            <label for="i2" class="next">▷</label>
+        </div>
+
+        <div class="slide-img" id="two">
+            <img src="https://images.nintendolife.com/f73ee7617a2d3/mutant-mudds-collection-switch.original.jpg"
+                alt="img2">
+            <span> Atooi Knocks 80% Off Select Switch Games, But The Sale Ends Soon <a
+                    href="https://www.nintendolife.com/news/2021/01/atooi_knocks_80_percent_off_select_switch_games_but_the_sale_ends_soon">
+                    Read Full Story >></a>
+            </span>
+            <label for="i1" class="prev">◁</label>
+            <label for="i3" class="next">▷</label>
+        </div>
+
+        <div class="slide-img" id="three">
+            <img src="https://gamespot1.cbsistatic.com/uploads/screen_kubrick/123/1239113/3785717-re_village_sept_screens_09.jpg"
+                alt="img3">
+            <span> Resident Evil Village Release Date Is May 7, PS4/Xbox One Versions Confirmed<a
+                    href="https://www.gamespot.com/articles/resident-evil-village-release-date-is-may-7-ps4-xbox-one-versions-confirmed/1100-6486601/">
+                    Read Full Story >></a>
+            </span>
+            <label for="i2" class="prev">◁</label>
+            <label for="i4" class="next">▷</label>
+        </div>
+
+        <div class="slide-img" id="four">
+            <img src="https://images.nintendolife.com/c2a27609accc8/super-mario-3d-world-bowsers-fury.original.jpg"
+                alt="img4">
+            <span> Super Mario 3D World + Bowser's Fury In These New Japanese Commercials<a
+                    href="https://www.nintendolife.com/news/2021/01/get_another_look_at_super_mario_3d_world_plus_bowsers_fury_in_these_new_japanese_commercials">
+                    Read Full Story >></a>
+            </span>
+            <label for="i3" class="prev">◁</label>
+            <label for="i5" class="next">▷</label>
+        </div>
+
+        <div class="slide-img" id="five">
+            <img src="https://images.nintendolife.com/9114f900f50f6/best-wii-u-games.original.jpg" alt="img5">
+            <span> Feature: Best Wii U Games<a href="https://www.nintendolife.com/guides/feature-best-wii-u-games">
+                    Read Full Story >></a></span>
+
+            <label for="i4" class="prev">◁</label>
+            <label for="i1" class="next">▷</label>
+        </div>
+    </div>
+    <!-- bottoms "dots" Funtionality -->
+    <div class="nav">
```

Comment:
Maybe a more descriptive class name would be nice here so you don't mix it up with the nav bar 😇